app/models/admin/status_batch_action.rb

```diff
@@ -31,7 +31,7 @@ def process_action!
     case type
     when 'delete'
       handle_delete!
-    when 'mark_as_sensitive'
+    when 'mark_as_sensitive', 'sensitive'
```

Comment:
The existing non-API controller uses `mark_as_sensitive` as the action "type". I wanted the new Status API to be consistent with the nomenclature of the equivalent Accounts API (which just uses `sensitive`), so I added another "type" that maps to the same action.